modules/ROOT/pages/clauses/create.adoc

```diff
@@ -3,247 +3,142 @@
 [[query-create]]
 = CREATE
 
-[abstract]
---
-The `CREATE` clause is used to create nodes and relationships.
---
+== Introduction
 
+The `CREATE` clause allows you to create nodes and relationships.
+Similar to xref::clauses/match.adoc[`MATCH`], this is done using xref::patterns/index.adoc[patterns].
+However, the patterns you can use in `CREATE` are restricted to what is relevant to describe the newly created nodes and relationships.
```

Comment:
I feel uneasy about this part.
You create elements by spelling out explicit nodes or paths: `(:A)`, `(:A)-[r:R]->({foo: "bar"})`.
You match on elements using nodes or path patterns. A path pattern represents the set of paths in the graph matching the pattern at the time of execution of the query.
The node `(:A)` is very different indeed from the node pattern `(:A)`, the lattern will also return node `(:A:B)` for example.
The syntax for a path happens to be valid syntax for a path pattern, but I'm not sure it is useful to present the path syntax as a restricted version of the path pattern syntax.
The same way the string "hello" is a valid regex that will match on the string "hello". When you ask the user for a string, you don't tell them to enter a restricted regular expression.
I know that we haven't made that distinction in our parser or in our AST[^1], but I think we should make it clearer to our users.

[^1]: I think we should differentiate it there as well but it's a different conversation.